lms/djangoapps/teams/api.py

```diff
@@ -50,3 +64,110 @@ def discussion_visible_by_user(discussion_id, user):
     """
     team = get_team_by_discussion(discussion_id)
     return not is_team_discussion_private(team) or user_is_a_team_member(user, team)
+
+
+def _is_course_admin(user, course_key):
+    """
+    Returns True if the user is an admin for the course, else returns False
+    """
+    if user.is_staff:
+        return True
+    if CourseStaffRole(course_key).has_user(user) or CourseInstructorRole(course_key).has_user(user):
```

Comment:
I think this might be just a miss. `CourseInstructorRole` is different from a `CourseStaffRole`. See https://github.com/edx/edx-platform/blob/master/common/djangoapps/student/roles.py#L261. I want to add it in to prevent the confusion later.